server/proxy/install.go

```diff
@@ -263,6 +263,21 @@ func (p *Proxy) ensureBotNamed(r *incoming.Request, app *apps.App, icon io.Reade
 		} else {
 			bot.UserId = user.Id
 			bot.Username = user.Username
+
+			if user.GetDisplayName(model.ShowFullName) != app.DisplayName {
```

Comment:
I'm thinking we can have a comment here like this:

```suggestion
			// The app has changed its display name
			if user.GetDisplayName(model.ShowFullName) != app.DisplayName {
```